charts/portworx/templates/portworx-rbac-config.yaml

```diff
@@ -23,13 +23,10 @@ rules:
     verbs: ["delete", "get", "list", "watch", "update"]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims", "persistentvolumes"]
-    verbs: ["get", "list", "create", "delete", "update"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["storageclasses", "csinodes"]
     verbs: ["get", "list"]
   - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments"]
-    verbs: ["get", "list", "create", "delete", "update"]
```

Comment:
Required by the CSI cloud drives

Reply:
Same here -- this should now be in sync with the install.portworx.com

E.g. https://install.portworx.com/?mc=false&kbver=1.19.3&k=etcd%3Ahttp%3A%2F%2F70.0.95.189%3A2379&m=eth1&d=eth1&c=px-cluster-123&stork=true&st=k8s&csi=true
```yaml
- apiGroups: ["storage.k8s.io"]
  resources: ["storageclasses"]
  verbs: ["get", "list"]
```